playbooks/common/openshift-etcd/scaleup.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Configure etcd
+  hosts: oo_new_etcd_to_config
+  serial: 1
+  any_errors_fatal: true
+  vars:
+    etcd_ca_host: "{{ groups.oo_etcd_to_config.0 }}"
+    etcd_peer_url_scheme: https
+    etcd_peer_port: 2380
+  pre_tasks:
+  - name: Add new etcd members to cluster
+    command: >
+      /usr/bin/etcdctl  --cert-file /etc/etcd/peer.crt
```

Comment:
Actually:
```yaml
TASK [Add new etcd members to cluster] *****************************************
fatal: [10.8.174.254 -> None]: FAILED! => {
    "changed": true, 
    "cmd": [
        "/usr/bin/etcdctl", 
        "--cert-file", 
        "/etc/etcd/peer.crt", 
        "--key-file", 
        "/etc/etcd/peer.key", 
        "--ca-file", 
        "/etc/etcd/ca.crt", 
        "-C", 
        "https://10.8.175.78:2379", 
        "member", 
        "add", 
        "10.8.174.254", 
        "https://172.16.186.31:2380"
    ], 
    "delta": "0:00:00.021797", 
    "end": "2017-05-04 06:56:48.534822", 
    "failed": true, 
    "rc": 1, 
    "start": "2017-05-04 06:56:48.513025", 
    "warnings": []
}

STDERR:

2017-05-04 06:56:48.524513 I | warning: ignoring ServerName for user-provided CA for backwards compatibility is deprecated
client: etcd cluster is unavailable or misconfigured; error #0: x509: certificate is valid for 172.16.186.182, not 10.8.175.78
```

as `/etc/etcd` directory does not exist yet. Since the ``openshift_etcd`` role that is run later creates the directory and its dependencies the certs.